bugalink-frontend/pages/login/index.tsx

```diff
@@ -28,15 +39,42 @@ export default function Login() {
   ) => {
     e.preventDefault();
     setIsLoading(true);
-    try {
-      await signIn('credentials', {
-        email,
-        password,
-        redirect: true,
-        callbackUrl: '/',
-      });
-    } catch (error) {
-      setIsLoading(false);
+
+    if (formRef.current) {
+      const formData = new FormData(formRef.current);
+      const values: FormValues = {
+        email: formData.get('email') as string,
+        password: formData.get('password') as string,
+      };
+
+      const errors: FormErrors = {};
+      const emailRegex = /^\S+@\S+$/i;
+      if (!email) {
+        errors.email = 'Por favor, ingrese su correo electrónico';
+      } else if (!emailRegex.test(email)) {
+        errors.email = 'Por favor, ingrese un correo electrónico válido';
+      }
+      
+      if (!password) {
+        errors.password = 'Por favor, introduce una contraseña';
```

Comment:
Change the verb conjugation so that it is also "usted" as with all other validations and please resolve or ignore sonar cloud errors/warnings.